<launch>

 <node name="usb_cam" type="usb_cam_node" pkg="usb_cam">
  <param name="video_device" value="/dev/video1" type="string"/>
 </node>
</launch>
